--- /usr/m0n0wall/build84/tmp/mini_httpd-1.19/mini_httpd.c	2014-08-16 16:46:27.000000000 +0800
+++ mini_httpd.c	2014-08-16 17:38:54.000000000 +0800
@@ -1,6 +1,6 @@
 /* mini_httpd - small HTTP server
 **
-** Copyright © 1999,2000 by Jef Poskanzer <jef@mail.acme.com>.
+** Copyright ? 1999,2000 by Jef Poskanzer <jef@mail.acme.com>.
 ** All rights reserved.
 **
 ** Redistribution and use in source and binary forms, with or without
@@ -1407,7 +1407,7 @@
 		captive_reqpath = path;
 		
 		if (cpelementpath != NULL && cpelementhost != NULL &&
-				host != NULL && strcmp(cpelementhost, host) == 0) {
+				host != NULL /*&& strcmp(cpelementhost, host) == 0 */) {
 			/* the host name in the request headers matches our host name;
 			   see if the request matches a CP element */
 			char *mypath, *myfile;
